ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -940,7 +954,7 @@ def runTest(self):
             for ip in sorted(self.logs_info.keys()):
                 self.log("Extracted log info from %s" % ip)
                 for msg in sorted(self.logs_info[ip].keys()):
-                    if msg != 'error':
+                    if not msg in [ 'error', 'route_timeout' ]:
```

Comment:
What aobut other errors. Should we check for them?

Reply:
Errors and route_timeout takes else branch to get printed in a different format.